bugs.rst

```diff
@@ -233,7 +245,7 @@ ISPC:
 ====================================
 Full list of ISPC bugs can be found `here <https://github.com/ispc/ispc/issues?q=is%3Aissue+label%3Ayarpgen+>`_
 
-| `1851 <https://github.com/ispc/ispc/issues/1851>`_ - LLVM assertion `Def == PreviousDef' failed.
+| `1851 <https://github.com/ispc/ispc/issues/1851>`_ - LLVM assertion \`Def == PreviousDef' failed.
```

Comment:
Is this an intentional change?

Reply:
Yes, otherwise " \` " symbol is processed by MarkDown.